File: Day 1/FizzBuzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let result = [];
    for (let i=1; i<n+1; i+=1){

    const divisibleBy3 = (i % 3 === 0);
    const divisibleBy5 = (i % 5 === 0);

    if (divisibleBy3 && divisibleBy5){
        result.push("FizzBuzz");
    }
    else if (divisibleBy3){
        result.push("Fizz");
    }
    else if (divisibleBy5){
        result.push("Buzz");
    }
    else {
        result.push(i.toString());
    }}
    return result;
};

File: Day 1/FizzBuzz.py
from typing import List


class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        result = []

        for i in range(1, n + 1):

            divisibleBy3 = (i % 3 == 0)
            divisibleBy5 = (i % 5 == 0)

            if divisibleBy3 and divisibleBy5:
                result.append("FizzBuzz")
            elif divisibleBy3:
                result.append("Fizz")
            elif divisibleBy5:
                result.append("Buzz")
            else:
                result.append(str(i))

        return result
